mitosheet/mitosheet/step_performers/pivot.py

```diff
@@ -369,6 +373,13 @@ def _execute_pivot(
         was_series = True
 
     if flatten_column_headers:
+
+        # Get the correct version of flatten column header
+        if public_interface_version == 1:
+            from mitosheet.public.v1 import flatten_column_header
+        else:
```

Comment:
If we're going to add ```raise Exception(f'Please add support for public_interface_version={public_interface_version}')``` in one place, we should probably add it in all of them cause we're going to rely on it.